lib/blockchain_double.js

```diff
@@ -534,13 +527,8 @@ BlockchainDouble.prototype.processCall = function(tx, blockNumber, callback) {
       if (err) {
         return callback(err);
       }
-      stateManager.getStateRoot((err, startingStateRoot) => {
```

Comment:
we don't need to get the current state root (which we were only using so we could put it back again later) because we are just going to throw this whole VM away now.